css/src/components/modal.css

```diff
@@ -71,17 +95,33 @@
 
 .Modal-footer {
   display: flex;
+  align-items: center;
+  height: 80px;
```

Comment:
No need to pass `height`

Reply:
Height of footer is fixed as per design guidelines.